drivers/serial/uart_stm32.c

```diff
@@ -172,7 +172,7 @@ static void uart_stm32_irq_err_enable(struct device *dev)
 	/* Enable FE, ORE interruptions */
 	LL_USART_EnableIT_ERROR(UartInstance);
 	/* Enable Line break detection */
-#ifndef CONFIG_SOC_STM32F030X8
+#if !(defined(CONFIG_SOC_STM32F030X8) || defined(CONFIG_SOC_STM32F091XC))
```

Comment:
How about we do #ifndef CONFIG_SOC_SERIES_STM32F0X.  So we don't have to keep updating the list for each F0 SoC that gets supported.

Reply:
Done.

Actually FOx SoCs support the LIN break detection but the driver does not support it currently.